lib/streamlit/net_util.py

```diff
@@ -43,8 +44,13 @@ def get_external_ip() -> Optional[str]:
     if _external_ip is not None:
         return _external_ip
 
+    # try to get external IP from Amazon http url
     response = _make_blocking_http_get(_AWS_CHECK_IP, timeout=5)
 
+    # try to get external IP from Amazon https url
```

Comment:
nit: I feel like the code here is straightforward enough that the comments aren't needed